Components/ShopConfigUpdater.php

```diff
@@ -26,30 +30,37 @@ class ShopConfigUpdater
      */
     private $shopConfig;
 
-    public function __construct(Connection $connection, ShopConfig $shopConfig)
+    /**
+     * @var ModelManager
+     */
+    private $modelManager;
+
+    public function __construct(Connection $connection, ShopConfig $shopConfig, ModelManager $modelManager)
     {
         $this->connection = $connection;
         $this->shopConfig = $shopConfig;
+        $this->modelManager = $modelManager;
     }
 
     /**
      * @param float $oldTaxRate
```

Comment:
these annotations should be removed. doesn't the cs-fixer do that? :thinking:

Reply:
Done